sinks/signalfx/signalfx.go

```diff
@@ -280,3 +249,86 @@ func (sfx *SignalFxSink) SetExcludedTags(excludes []string) {
 	}
 	sfx.excludedTags = tagsSet
 }
+
+type ddSampleKind int
+
+const (
+	ddSampleEvent ddSampleKind = iota
+	ddSampleServiceCheck
+	ddSampleUnknown
+)
+
+func ddOtherSampleKind(tags map[string]string) ddSampleKind {
+	if _, ok := tags[dogstatsd.CheckIdentifierKey]; ok {
+		return ddSampleServiceCheck
+	}
+	if _, ok := tags[dogstatsd.EventIdentifierKey]; ok {
+		return ddSampleEvent
+	}
+	return ddSampleUnknown
+}
+
+func (sfx *SignalFxSink) convertServiceCheck(sample *ssf.SSFSample) error {
```

Comment:
In line with the above thing about `reportEvent`, I think either this should be called `reportServiceCheck`.

Reply:
Okay incorporating naming change.